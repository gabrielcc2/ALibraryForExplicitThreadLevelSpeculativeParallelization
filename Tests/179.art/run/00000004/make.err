scanner.c: In function ‘analog_conv’:
scanner.c:287:4: warning: format ‘%f’ expects type ‘float *’, but argument 4 has type ‘double *’
scanner.c:287:4: warning: format ‘%f’ expects type ‘float *’, but argument 6 has type ‘double *’
scanner.c: In function ‘get_pat’:
scanner.c:299:6: warning: format ‘%f’ expects type ‘float *’, but argument 3 has type ‘double *’
scanner.c: In function ‘load_weights’:
scanner.c:780:11: warning: ignoring return value of ‘fscanf’, declared with attribute warn_unused_result
scanner.c:788:14: warning: ignoring return value of ‘fscanf’, declared with attribute warn_unused_result
scanner.c: In function ‘get_pat’:
scanner.c:299:12: warning: ignoring return value of ‘fscanf’, declared with attribute warn_unused_result
scanner.c: In function ‘analog_conv’:
scanner.c:285:10: warning: ignoring return value of ‘fscanf’, declared with attribute warn_unused_result
scanner.c:287:14: warning: ignoring return value of ‘fscanf’, declared with attribute warn_unused_result
