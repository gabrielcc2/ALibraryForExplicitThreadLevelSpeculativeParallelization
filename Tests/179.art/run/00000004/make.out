gcc -c -o scanner.o            -O3    scanner.c
gcc     -O3    scanner.o   -lm  -o art
